src/toil/test/provisioners/gceProvisionerTest.py

```diff
@@ -0,0 +1,338 @@
+# Copyright (C) 2015 UCSC Computational Genomics Lab
```

Comment:
Should this be combined with the AWS provisioner class to make an `AbstractAbstractProvisionerTest`? (haha)

Reply:
Yes. I tried, but it is not simple. We can tackle this when we do a refactor after the Azure provisioner is done.